### ISSUE TITLE
[Bug][a11y] Empty state module : Usage label and buttons not grouped

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ods-ios#700]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Plus

### OS Version

17.1.1

### App version

v0.18.0

### Describe the issue
related to #558 
On module "Empty State", "usage" label and associated buttons are not grouped, so when Voice Over focus moves to one of the button, only the button is vocalized, not the group label. 

### Expected Behavior

Usage label and associated buttons should be grouped together to have an optimal vocalization when focus first moves to a button of the list.
Try using : `.accessibilityElement(children: .contain)`

**Comment by ludovic35** [comment]
Fixed by #677

**Comment by pya35** [comment]
fixed
